test/parallel/test-readline-promises-interface.js

```diff
@@ -952,6 +952,24 @@ for (let i = 0; i < 12; i++) {
     rli.close();
   }
 
+  // Call question after close
+  {
+    const [rli, fi] = getInterface({ terminal });
+    rli.question('What\'s your name?').then(common.mustCall((name) => {
+      assert.strictEqual(name, 'Node.js');
+      rli.close();
+      rli.question('How are you?')
+        .then(common.mustNotCall())
+        .catch(common.expectsError({
```

Comment:
I think we want to use only one `.then` call otherwise the error thrown by `common.mustNotCall` will be catched by the `catch` block (which should still make the test fail, but it's cleaner to assert exactly what we expect imo)
```suggestion
        .then(common.mustNotCall(), common.expectsError({
```